drivers/driver.go

```diff
@@ -211,8 +234,11 @@ func (r RDMAVolumeDriver) Mount(request volume.MountRequest) volume.Response {
 	// Ensure the r is properly confiured
 	r.validateOrCrash()
 
+	// Pass the mount request to the volume database.
+	mountpoint, err := r.VolumeDatabase.Mount(request.Name, request.ID)
+
 	// Pass the mount request to the storage controller.
-	mountpoint, err := r.StorageController.Mount(request.Name, request.ID)
+	// TODO mountpoint, err := r.StorageController.Mount(request.Name, request.ID)
```

Comment:
TODOs should be made into issues and linked to the issue in the code comment with at least a #number. Otherwise, they should be removed.